index.html

```diff
@@ -307,10 +307,10 @@ <h4 class="card__header">Wojtek</h4>
                         <a href="#" class="card__link" target="_blank">
```

Comment:
please add https://twitter.com/wojtrawi?lang=pl

Reply:
Should work now, please re-check